### DISCUSSION TITLE
deduplication EUDASM - ISRIC Library

### DISCUSSION THREAD

**Discussion by pvgenuchten** [soilwise-he/harvesters#93] [Ideas]
[EUDASM](https://esdac.jrc.ec.europa.eu/resource-type/national-soil-maps-eudasm) (2.500 maps) is a set of scanned maps from the ISRIC collection hosted by ESDAC.
A large portion of those maps is also made available through [ISRIC library](https://www.isric.org/explore/library) (10.000 maps).
Would be nice to understand from deduplication how much is the overlap.
For that we would need to harvest the ISRIC library

ISRIC library can be queried in [CSV format](https://library.wur.nl/WebQuery/isric/isric.csv?q=hydrology%20nigeria&wq_srt_desc=score&wq_max=250)

Harvest code is available at https://github.com/soilwise-he/harvesters/tree/main/isric-library

**Comment by BerkvensNick** [comment]
@pvgenuchten , I can see that there are EUDASM records in the catalogue (4748 in total if I use it as a keyword to search on), but not sue how using the API I can get hold of these records, was looking at https://soilwise-he.containers.wur.nl/cat/openapi, but not really sure how I can use "EUDASM" as a keyword to search on?

**Comment by pvgenuchten** [comment]
True, we currently do not have that property yet on the filters

**Reply by BerkvensNick** [reply]
is there a way I can get hold of these EUDASM records in the catalogue to examine how duplicates can be identified coming from both sources? Or should I get them from ESDAC (and if so how did you harvest them there)?
